app/services/moves/finder.rb

```diff
@@ -27,6 +28,14 @@ def apply_filters(scope)
       scope
     end
 
+    def apply_active_filter(scope)
+      if filter_params.key?(:active)
+        scope.merge(Move.active)
+      else
+        scope
+      end
+    end
+
```

Comment:
This way to go is starting to be a bit "smelly", this could have been a chance to improve things a bit here.